.github/workflows/build.yaml

```diff
@@ -44,27 +44,30 @@ jobs:
         uses: actions/cache@v4
         with:
           path: data/geog
-          key: ${{ runner.os }}
+          key: geog-data-${{ runner.os }}
       - name: Download and extract geography data
         if: steps.cache-geog.outputs.cache-hit != 'true'
         run: |
           mkdir -p data/geog
-          curl "https://www2.mmm.ucar.edu/wrf/src/wps_files/geog_low_res_mandatory.tar.gz" -o data/geog/geog_low_res_mandatory.tar.gz
-          tar -xvzf data/geog/geog_low_res_mandatory.tar.gz -C data/geog
+          wget -qN https://www2.mmm.ucar.edu/wrf/src/wps_files/geog_low_res_mandatory.tar.gz -P data/geog
+          wget -qN https://www2.mmm.ucar.edu/wrf/src/wps_files/landuse_10m.tar.bz2 -P data/geog
+          tar -xf data/geog/geog_low_res_mandatory.tar.gz -C data/geog
+          tar -xf data/geog/landuse_10m.tar.bz2 -C data/geog/WPS_GEOG_LOW_RES
+          mv data/geog/WPS_GEOG_LOW_RES data/geog/WPS_GEOG
```

Comment:
This looks (nearly) identical to the `data/geog` in the Makefile, is it worth invoking `make data/geog` here instead of duplicating?

Alternately, we could put this whole block into a `download-geog.sh` script and invoke that in both places. One small advantage of that approach would be that we could use a hash of the file as the cache `key` in `actions/cache@v4`. This way if we ever change the script contents (URLs, etc) it would invalidate the GHA cache and run it again.

```yaml
  key: geog-data-${{ runner.os }}-${{ hashFiles('download-geog.sh') }}
```

Reply:
Nearly the same, but not quite. The GHA can only use the low res datasets as the default geog data is 30GB uncompressed which is too big for the cache.

Good idea about hashing the download scripts. I'll add in something that can fetching low/high resolution data

Reply:
@aethr Added a download script `scripts/download-geog.sh`

Reply:
Love it, nice work.